### DISCUSSION TITLE
splendidsuperdlscord ban appeal of minecolonies discord sever

### DISCUSSION THREAD

**Discussion by SplendidSuperGithub** [ldtteam/minecolonies#9281] [General]
i have time other account, this accoount was banned i was banned for advertising a 18+ discord sever, and yeah i advertise this server, so you will ask me, why i un ban me of the oficial discord if i doed this? me with this other account doed this becuase someone have access to this account and i didin't know my account controlled by other person was spamming thoose links, so yeah, i have been hacked and that's the why i sended thoose links, i promise i will be carefull to don't be hacked again

**Comment by MotionlessTrain** [comment]
Once a discord account is hacked, it is considered a risk forever, which is why it has been banned.

You are free to make a new discord account and join the discord server with it

**Reply by SplendidSuperGithub** [reply]
i'm using a new account i deleted the past account the new account of discord is splendidsuperdlscord, so this is a new account, account was hacked was deleted this is a new account, but i need to have a un ban of my ip becuase i can't enter inside minecolonies discord because the banned my ip

**Reply by MotionlessTrain** [reply]
Your account got banned from the discord server, not your ip (we can't do that, even)

If for some reason your ip is blocked, then that is discord itself doing so, and you need to contact them

**Reply by SplendidSuperGithub** [reply]
Ok, i try with discord invite links of minecolonies discord, but say me is invalid or expired, can you send me here a invite link of discord sever of minecolonies

**Reply by MotionlessTrain** [reply]
https://discord.minecolonies.com

**Reply by SplendidSuperGithub** [reply]
Say me the same, invalid or expired. I think would work if you create a new invite link and don’t copy any link have existing a lot of time
